doc/developer-notes.md

```diff
@@ -91,6 +91,9 @@ code.
   - Compile-time constant names are all uppercase, and use `_` to separate words.
   - Class names, function names, and method names are UpperCamelCase
     (PascalCase). Do not prefix class names with `C`.
+  - Enumerators may be snake_case, PascalCase, or ALL_CAPS. This is a more tolerant policy than
+    the [C++ Core Guidelines](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Renum-caps).
```

Comment:
In commit "doc: update enum naming in developer notes" (2d813c42ed5573d3afd80d677e2ded41e8d5b7e8)

Would be nice to add

> , which recommend using `snake_case`

to the end of this sentence to save the trouble of having to click the link to see what is suggested

Reply:
done